force-app/main/default/classes/RD2_EntryFormController.cls

```diff
@@ -664,4 +677,85 @@ public with sharing class RD2_EntryFormController {
             }
         }
     }
+
+    public class RD2SaveRequest {
+        @AuraEnabled public Id recordId;
+        @AuraEnabled public String recordName;
+        @AuraEnabled public String recurringStatus;
+        @AuraEnabled public Id contactId;
+        @AuraEnabled public Id accountId;
+        @AuraEnabled public Date dateEstablished;
+        @AuraEnabled public Decimal donationValue;
+        @AuraEnabled public String currencyIsoCode;
+        @AuraEnabled public String recurringPeriod;
+        @AuraEnabled public Integer recurringFrequency;
+        @AuraEnabled public Date startDate;
+        @AuraEnabled public String dayOfMonth;
+        @AuraEnabled public Integer plannedInstallments;
+        @AuraEnabled public String recurringType;
+        @AuraEnabled public String paymentToken;
+        @AuraEnabled public String commitmentId;
+        @AuraEnabled public String paymentMethod;
+        @AuraEnabled public Map<String, Object> customFieldValues;
+
+        public npe03__Recurring_Donation__c toRecord() {
+            npe03__Recurring_Donation__c rd = new npe03__Recurring_Donation__c();
```

Comment:
The Campaign field is missing here

Reply:
Also for Edit Entry form, we have more fields such as Status that will be displayed onto the form

Reply:
and `rd.ClosedReason__c` as well